website_and_docs/content/documentation/grid/configuration/cli_options.en.md

```diff
@@ -349,3 +350,92 @@ driver.quit();
 ```
 
 Set the custom capability to `false` in order to match the Node B.
+
+#### Specifying path from where downloaded files can be retrieved
+
+At times a test may need to access files that were downloaded by it on the Node. To retrieve such files, following can be done.
+
+##### Start the Hub
+```
+java -jar selenium-server-<version>.jar hub
+```
+
+##### Start the Node with downloads path specified
+```
+java -jar selenium-server-<version>.jar node --downloads-path /usr/downloads
+```
+
+##### Sample that retrieves the downloaded file
+
+```java
+import static org.openqa.selenium.remote.http.Contents.string;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.io.Zip;
+import org.openqa.selenium.json.Json;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.remote.http.HttpClient;
+import org.openqa.selenium.remote.http.HttpMethod;
+import org.openqa.selenium.remote.http.HttpRequest;
+import org.openqa.selenium.remote.http.HttpResponse;
+
+public class DownloadsSample {
+
+  public static void main(String[] args) throws InterruptedException, IOException {
+    File dirToCopyTo = new File("/usr/downloads");
+    URL gridUrl = new URL("http://localhost:4444");
+    RemoteWebDriver driver = new RemoteWebDriver(gridUrl, firefoxOptions());
+    driver.get("http://the-internet.herokuapp.com/download");
+    WebElement element = driver.findElement(By.cssSelector(".example a"));
+    element.click();
+
+    TimeUnit.SECONDS.sleep(10);
```

Comment:
i would add a wait condition here to see that 1 file was added to the download location folder instead of sleep.

Reply:
@nirtal85 - The file download happens on a remote node and the test may be running locally. So not sure how this wait is going to look like.